.travis.yml

```diff
@@ -4,8 +4,8 @@ language: python
 # support minimum python versions specified here:
 # https://docs.ansible.com/ansible/dev_guide/developing_modules_python3.html
 python:
-  - "2.6"
-  - "3.5"
+  - 2.7
+  - 3.5
```

Comment:
It seems that something automated mangled the Travis YAML file, let's undo these unnecessary changes. I specifically do this so that weird floating-point errors don't occur, YAML has a lot more types and a lot more implicit conversions than, say JSON.